ping.go

```diff
@@ -330,7 +330,7 @@ func (p *Pinger) Run() {
 
 	err = p.sendICMP(conn)
```

Comment:
Adding a nolint here should fix the tests for now.

```suggestion
	err = p.sendICMP(conn) //nolint:errcheck
```